libs/badge-list.ts

```diff
@@ -74,6 +74,7 @@ export const liveBadgeList = [
   'liberapay',
   'runkit',
   'https',
+  'open-vsx',
```

Comment:
Suggestion: place it near other extension stores to keep the list organized :bulb: